src/Compilers/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -42,4 +42,5 @@ virtual Microsoft.CodeAnalysis.Operations.OperationVisitor.VisitTupleBinaryOpera
 virtual Microsoft.CodeAnalysis.Operations.OperationVisitor<TArgument, TResult>.VisitConstructorBodyOperation(Microsoft.CodeAnalysis.Operations.IConstructorBodyOperation operation, TArgument argument) -> TResult
 virtual Microsoft.CodeAnalysis.Operations.OperationVisitor<TArgument, TResult>.VisitDiscardOperation(Microsoft.CodeAnalysis.Operations.IDiscardOperation operation, TArgument argument) -> TResult
 virtual Microsoft.CodeAnalysis.Operations.OperationVisitor<TArgument, TResult>.VisitMethodBodyOperation(Microsoft.CodeAnalysis.Operations.IMethodBodyOperation operation, TArgument argument) -> TResult
-virtual Microsoft.CodeAnalysis.Operations.OperationVisitor<TArgument, TResult>.VisitTupleBinaryOperator(Microsoft.CodeAnalysis.Operations.ITupleBinaryOperation operation, TArgument argument) -> TResult
\ No newline at end of file
+virtual Microsoft.CodeAnalysis.Operations.OperationVisitor<TArgument, TResult>.VisitTupleBinaryOperator(Microsoft.CodeAnalysis.Operations.ITupleBinaryOperation operation, TArgument argument) -> TResult
+Microsoft.CodeAnalysis.Diagnostics.OperationAnalysisContext.SemanticModel.get -> Microsoft.CodeAnalysis.SemanticModel
```

Comment:
@sharwell  How is ^supposed to work, tried editing manually but dumbed whitespace somehow. Not sure I messed with the right file.

Reply:
you shouldn't have to manually edit.  if you're in roslyn, you can just bring up the quick-fix on the locatoin of the error and it will have an option to update this file for you.

Reply:
Ah, ok, tooling was pretty broken for me, did not even try to build. I ran VS in slow notepad mode. :)